### ISSUE TITLE
[FEATURE] add an output flag in the cli

### ISSUE THREAD

**Issue by yapudjus** [nathom/streamrip#246]
### Is the feature request related to a problem. Please describe it.

i am creating a script to sync my downloads with my deezer account but i can't find a cli argument to specify the download folder

### Describe the solution you would like.

add an argument to it's cli, for example 
```rip url --out /media/yapudjus/6cd7d80f-945e-42de-8713-fb6eec68d0e2/artists/ --codec mp3 --max-quality 1 https://www.deezer.com/us/artist/1001```

### Describe alternatives you've considered.

i currently limit myself to only downloading the artists, and on windows i leave blank the `folder = ""` line and put a `cd {path} &&` before my download command, but i would like an easier solution, and that also work on linux, such as an `--out` flag

**Comment by bianjunjie1981** [comment]
The configuration file supports the configuration download folder.  line3 `folder = `
I think it is better to add a parameter mode, 
such as `--config.***` is equivalent to `*** = ` in the configuration file, 
which is used to override the configuration value of the configuration file this runtime, 
so that it has the highest flexibility.

**Comment by yapudjus** [comment]
i changed my code from this

```
def deezer_down(url, title='', quality='3', codec='flac', path='.'):
    # os.system('clear')
    os.system(f'xtitle {title}')
    command = f'cd {path} && rip url --ignore-db --codec {codec} --max-quality {quality} {url}'
    print(command)
    os.system(command)
    print(f'done with {title}')
```
to this

```
def deezer_down(url, title='', quality='3', codec='flac', path='.'):
    # os.system('clear')
    os.system(f'xtitle {title}')
    with open(config_path, 'r') as file: data = file.readlines()
    print(f'changing path in {config_path} to\nfolder = "{path}"')
    data[2] = str(f'folder = "{path}"\n')
    with open(config_path, 'w') as file: file.writelines(data)
    command = f'rip url --ignore-db --codec {codec} --max-quality {quality} {url}'
    print(command)
    os.system(command)
    print(f'done with {title}')
```

and now it works, can't start multiple instances but at least it work as a temporary solution

**Comment by nathom** [comment]
Implemented in 6719676b6e5bd00ae5b5027318f006792be29779. Thanks for the suggestion.
